core/misc/backdrop.js

```diff
@@ -486,4 +486,29 @@ Backdrop.theme.prototype = {
   }
 };
 
+/**
+ * Add helper functions for feature detection
+ */
+Backdrop.featureDetect = {};
```

Comment:
@quicksketch what do you think of this? Wanted to add some extendable way to feature test?